librz/core/cbin.c

```diff
@@ -632,11 +632,7 @@ RZ_API bool rz_core_bin_apply_dwarf(RzCore *core, RzBinFile *binfile) {
 		return false;
 	}
 
-	RzBinDWARFOption opt = {
-		.line_mask = RZ_BIN_DWARF_LINE_INFO_MASK_LINES,
-		.flags = RZ_BIN_DWARF_ALL - RZ_BIN_DWARF_LOC
-	};
-	RzBinDWARF *dw = rz_bin_dwarf_from_file(binfile, &opt);
```

Comment:
Why removing these?

Reply:
Because I found that the sections of DWARF are interdependent, and if you want to read a certain section, you often have to read several other sections at the same time, so I deleted this control option.